src/types/target.spec.ts

```diff
@@ -335,6 +336,20 @@ describe('Target', () => {
     })
   })
 
+  it('should include isDefault in JSON', () => {
+    const target = new Target({
+      name: 'test',
+      serverUrl: '',
+      serverType: ServerType.SasViya,
+      appLoc: '/test',
+      isDefault: true
```

Comment:
I'm not sure about this - what happens if there are multiple targets with 'isDefault': true ?

Wouldn't it be better to have a global setting, where you can choose the default branch?  Then there can really only be one, and if there's a change, it's only one place to change